components/azure-storage/setup-storage-event-grid.sh

```diff
@@ -19,7 +19,7 @@ az storage queue create \
   --only-show-errors \
   -o tsv >>log.txt
 
-storage_id=$(az storage account show --name $AZURE_STORAGE_ACCOUNT_GEN2 --query id -o tsv)
+storage_id=$(az storage account show --name $AZURE_STORAGE_ACCOUNT_GEN2 -g $RESOURCE_GROUP --query id -o tsv)
```

Comment:
It shouldn't because it explicitly uses the resource group just created by the script. Removing `-g $RESOURCE_GROUP`  will revert to the user's `az config set defaults` resource group which will throw an error if they've set one in the past. I recommend we leave it.

Reply:
That said we could submit it as an enhancement in future PR